Sources/IMADAIPlugin.swift

```diff
@@ -455,39 +453,60 @@ import PlayKitUtils
                 }
             }
         case .LOADED:
-            let adEvent = event.ad != nil ? AdEvent.AdLoaded(adInfo: PKAdInfo(ad: event.ad,
-                                                                              podCount: nil,
-                                                                              adPlayHead: .nan)) : AdEvent.AdLoaded()
+            var adEvent = AdEvent.AdLoaded()
```

Comment:
@chausov  I think we can move it to a method

Reply:
we can not wrap it to a method, because in each case there are different `AdEvent` created
in total 6 different methods needed, it will not save any lines of code: 
```Swift
AdEvent.AdLoaded()
AdEvent.AdStarted()
AdEvent.AdPaused()
AdEvent.AdResumed()
AdEvent.AdClicked()
```